.golangci.yaml

```diff
@@ -140,7 +140,7 @@ linters:
     - testifylint
     - unused
     - whitespace
-    - wrapcheck
+    # - wrapcheck
```

Comment:
If we disable it, why not just delete?  Or please put a comment explaining why it is commented :)

Reply:
Honestly I don't remember why I commented the line. I leave it to keep the list of the linters on the file, but the reference before the `enable:` section is enough.

I have removed the line.